### DISCUSSION TITLE
App not working on Android Tablet

### DISCUSSION THREAD

**Discussion by adrianeb2015** [fhdm-dev/scoppy#137] [Q&A] [answered]
The app works perfect on my phone, but it refuses to install on my Android 10 Tablet, let me know if you need more info to help to install it
Thanks!

**Comment by fhdm-dev** [comment, marked as answer]
Hi,
The only reason I can think of is that the tablet doesn't support USB OTG. What tablet are you using? And do you get an error message when you try and install?

**Comment by adrianeb2015** [comment]
Thanks for the quick answer, even at the Google Play store the apps shows its not compatible, The tablet is a Noga NGA 520, and it supports USB-OTG. 
![Capture](https://github.com/fhdm-dev/scoppy/assets/11236513/31f974ae-5faf-4125-a06d-0f6d3c0bc92f)

Update: Hi-Fi Cast - Music Player shows it works on the tablet
